.github/workflows/tests.yaml

```
@@ -26,6 +26,8 @@ jobs:
           - 2.5
           - 2.6
           - 2.7
+          - '3.0'
+          - 3.1
```

Maybe just to avoid parsing issue maybe we should quote all future versions with quotes?
```suggestion
          - '3.1'
```

---

hmm, the problem only occurs with 'x.0', so we can leave this as it is.
But either way is fine with me!